cardano-testnet/test/Test/Cli/Alonzo/LeadershipSchedule.hs

```diff
@@ -42,15 +42,15 @@ import qualified System.Directory as IO
 import           System.Environment (getEnvironment)
```

Comment:
So the question is should we bother testing the Alonzo leadership schedule? I can't think of a reason why we should. Did consensus ever give us feedback here?

Reply:
I think I won't get an answer before the New Year, so let's put that in a separate PR.

Reply:
I think I won't get an answer before the New Year, so let's put that in a separate PR.